### DISCUSSION TITLE
Añadir un estado "deprecado" en los Cover de proyecto en Figma

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1387] [Proposals]
### Goal

El objetivo principal el eliminar el uso incorrecto de las tags de estado que se hace para poder documentar que un flujo ha sido deprecado ya que actualmente se usa otro estado diferente al que se cambia el texto para indicar el estado deprecado

<img width="551" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/013c1a3d-6cb9-46a3-960d-e8c8e2f32f9e">

### Context

Lo indicado arriba

### Impact

Al generar este estado podemos estandarizar este estado en todos los flujos y que en futuras evoluciones del cover no se rompa.

### Discussion Questions

-

### Additional considerations

-

### Do you have a visual proposal?
https://www.figma.com/file/OJzyf3I6GpCPz4TdY0ilRW/Start-Tab-MovEs-UI?type=design&node-id=255-8214&mode=design&t=djBLCMilmhw58fFz-4

**Comment by yceballost** [comment]
lo añado como tarea y lo sacamos para la siguiente release
